app/components/Tooltip/icons/UserIcon.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default () => (
+  <span
+    // eslint-disable-next-line react/no-danger
+    dangerouslySetInnerHTML={{
```

Comment:
Is there a particular reason for using this method to display the icon instead of a svg file which is in use in other places in the project?

Reply:
Not really, I think I've just tunnel visioned here. I've copied this icon from another project. I'll fix it.